src/Sweep.cpp

```diff
@@ -103,7 +103,7 @@ void TravelDepartSweep(double t)
 						l = Airports[i].DestMcells[l].id;
 						int k = (int)(ranf_mt(tn) * ((double)Mcells[l].n));
 						int i2 = Mcells[l].members[k];
-						if ((abs(Hosts[i2].inf) < InfStat_InfectiousAsymptomaticNotCase) && (Hosts[i2].inf != InfStat_Case))
+						if (Hosts[i2].is_exposed() || Hosts[i2].is_susceptible())
```

Comment:
Check that this is a 1-1 with the original logic

Reply:
All of these are 1:1 with the original logic, to maintain the same logic. This was my primary goal in this PR. This means that sometimes things look a bit "nonsensical", but it's the best description of the underlying logic I could come up given what the old logic was.

As I said before, in refactoring the enum and simplifying the equations, I hope to come to better names as well in the future, to describe the underlying logic.